packages/cozy-mespapiers-lib/docs/AddPaperGuidelines.md

```diff
@@ -0,0 +1,15 @@
+# How to add a new paper:
+
+## You can add this checklist to your PR
+
+```
+- [ ] Look at the [papersDefinitions](./papersDefinitions) doc to respect the model.
```

Comment:
`First, get familiar with [papersDefinitions](./papersDefinitions) doc to be sure to understand correctly how it is built, and how to construct your new paper.`